### PR TITLE
Fixes detection of ETH balance on account page

### PR DESCRIPTION
**Pull request by manuelwedler**
To redirect the user to the swap page if they have enough ETH

### CODE REVIEW

**Comment by fredo** [comment]
how much is enough eth?

**Comment by manuelwedler** [comment]
https://github.com/raiden-network/raiden-wizard/blob/db86aeda4bfaed9901d180c71a9cfe80ac822670/resources/templates/base.html#L92
It is the amount that is set in the config file

File: resources/templates/account.html [hunk 1]
```diff
@@ -123,19 +125,37 @@ <h2>Fund your Raiden Account with ETH</h2>
    }
  }
 
+ function updateNeededEth(balance) {
+   neededEthAmount = ETHEREUM_REQUIRED_AMOUNT - balance.ETH.as_wei;
```
Comment (fredo) [inline resources/templates/account.html:129]: does js takes the outer scope variable automatically?

Comment (manuelwedler) [reply resources/templates/account.html:129]: Yes it does. Only if you would shadow the same variable name in the inner scope it would not.

**Pull request merged**